envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -381,26 +381,20 @@ class DynamicForwardProxyProperties {
 }
 
 class CompressionProperties {
-    var gzip = GzipProperties()
-    var brotli = BrotliProperties()
+    var gzip = CompressorProperties()
+    var brotli = CompressorProperties()
```

Comment:
how does chooseFirst work now that gzip and brotli use the same properties?

Reply:
so according to the docs if several filters have choose first then the priority of them will be chosen by their order in configuration, but I still going to configure as it was before but in aec settings